meetings/2023-08-23.md

```diff
@@ -0,0 +1,85 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2023-08-23
+
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=PkpPM9e5spM>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1428>
+
+## Present
+
+* Benjamin Gruenbaum @benjamingr (voting member)
+* Geoffrey Booth @GeoffreyBooth (voting member)
+* Gireesh Punathil @gireeshpunathil (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Darshan Sen @RaisinTen (voting member)
+* Richard Lau @richardlau (voting member)
+* Tobias Nießen @tniessen (voting member)
+* Claudio Wunder @ovflowd (Guest presenting on new WebSite design)
+
+## Agenda
+
+### Announcements
+
+* No announcements this week
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* Nominations for OpenJS CPC voting members #819
+
+### nodejs/node
+
+* esm: add import.meta.dirname and import.meta.filename [#48740](https://github.com/nodejs/node/pull/48740)
+  * Proposal opened on WinterCG site, but no response
+    * pinged both Bun and Deno, but no response.
+    * leave on agenda until next WinterCG meeting
+    * Michael suggested we should have separate issue about SemVer major or not for things in import.meta
+      * Geoffrey to open issue to discuss that.
+
+### nodejs/TSC
+
+* openjs-foundation/community-fund#36 to represent TSC to TC39.
+* Working as Champion on TC39 proposal related AsyncStorage proposal. Would like to be there in person to present stage 2 proposal.
+* 10 +1s to support requests on behalf of TSC from those in meeting and who chimed in through slack
+
+* Collaborator Meeting before the NodeConfEU #1422
+  * Discussed and agree last time, removing from agenda
+
+* Remaining OSSF Funding #1384
+  * Agreed we could remove from agenda
+
+### nodejs/admin
+
+* Nominations for OpenJS CPC voting members #819
+  * FYI given
+
+* Redesign of Node.js Website #818
+  * Claudio took us through the proposed layout and content.
+  * Tobias, was mentioned that Website team would not be responsible for content (in context of Learn content). So who would be responsible for this content?
+    * Claudio - was created out of past work from Website team, going forward teams across the project (any core collaborator) would own this content be able to managed/update.
+  * Fair amount of discussion on download page
+    * Richard -> Package manager are not controlled by project, should not be first choice
+    * Michael -> seems to miss option for binaries (prebuilt installer does not cover that)
+    * Tobias -> concern about recommend/show installing with package managers.
```

Comment:
```suggestion
    * Tobias -> concern about recommend/show installing with package managers, especially Chocolatey because we received negative feedback after adding it to the MSI
```